tests/compact/c015-in.jsonld

```diff
@@ -3,8 +3,14 @@
   "http://example/p": [{
     "@id": "http://example/typed-base#typed-id",
     "@type": ["http://example/Type"],
-    "http://example/nested": [{
-      "@id": "http://example/base-base#nested-id"
+    "http://example/subjectReference": [{
+      "@id": "http://example/typed-base#subject-reference-id"
+    }],
+    "http://example/nestedNode": [{
+      "@id": "http://example/base-base#nested-id",
+      "http://example/foo": [{
```

Comment:
Why should the nestedNode not use the base defined for `Type`? Nesting is/should be in-scope of the object containing the type; the purpose of nesting is to allow structure without affecting semantics, so values of a nested node should be treated as if they are part of the containing object on which `Type` is set.